src/components/WalletModal/PendingView.tsx

```diff
@@ -18,18 +17,16 @@ const PendingSection = styled.div`
   }
 `
 
-const StyledLoader = styled(Loader)`
-  margin-right: 1rem;
+const StyledLoader = styled.div`
```

Comment:
Can you either rename this `LoaderContainer` or keep the styles as extended onto `Loader`?